JsBasics/Conditions_and_loops/student_grades.js

```diff
@@ -0,0 +1,8 @@
+let averagegrade = 85
+if (averagegrade < 60) { console.log("Bed"); }
+
+else if (averagegrade > 60 && averagegrade <= 70) { console.log("Not bed"); }
+else if (averagegrade > 70 && averagegrade <= 80) { console.log("Good"); }
```

Comment:
Немає потреби додатково прописувати avaragegrade > 70
```suggestion
else if (averagegrade <= 80) { console.log("Good"); }
```